docs/benchmark.md

```diff
@@ -0,0 +1,98 @@
+# Benchmarks
+
+## Running the benchmark
+
+You can run the benchmark by executing `./gradlew :android-benchmark:benchmark:connectedCheck`. (Note that the benchmark should be run on an actual device)
```

Comment:
Put a newline after this if you want the next line to actually be a new line. Also let's make the Note just a second sentence

Reply:
bf05483